packages/rich-text-react-renderer/src/__test__/index.test.tsx

```diff
@@ -404,3 +404,55 @@ describe('nodeListToReactComponents', () => {
     expect(renderedNodes).toMatchSnapshot();
   });
 });
+
+describe.only('preserveWhitespace', () => {
```

Comment:
@YvesRijckaert should we remove the `only` 😅

Reply:
if you remove the only, what do you think about adding the new param also to the readme? :)

Reply:
Oh good catch! Did not see that  😅

Reply:
I wish the had commit failed 😬

Reply:
There is an eslint-rule to catch this things:
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-focused-tests.md
maybe we can activate this?